routers/repo/pull.go

```diff
@@ -928,6 +929,21 @@ func CleanUpPullRequest(ctx *context.Context) {
 		return
 	}
 
+	if err := repofiles.PushUpdate(
+		ctx.Repo.Repository,
```

Comment:
pr.HeadRepo ?

Reply:
yes, because on line 924 that is the var used when deleting (existing code)

if err := gitRepo.DeleteBranch(**pr.HeadBranch**, git.DeleteBranchOptions{

When I tested receiving the webhook it looked correct (to me)

Reply:
I mean maybe we should change `ctx.Repo.Repository` to `pr.HeadRepo` here?

Reply:
oops, I see what you mean now. I updated it